wireapp/kalium
# chore: temp local avs dependency until deployed to artifactory

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Currently, the AVS lib is not available on Sonatype, causing that the current workaround -creating a module for this dependency and adding it as a project- to break autocompletion and running tests on the IDE.

### Solutions

Point to a local -temp- m2 defined repo, leaving untouched the calling module. So when we bump to the proper version in artifactory, that one will be picked automatically over this local repo. Also added a repository filter, so we don't waste time in other project dependencies.

#### How to Test

Checkout the branch, and you should be able to build, and run specific tests on the project

#### Note

After the version is published on artifactory, we can remove the whole `avs` directory.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
